src/drive/web/modules/actions/index.jsx

```diff
@@ -75,21 +76,21 @@ export const download = ({ client, vaultClient }) => {
           )
         },
         action: files => exportFilesNative(client, files, { vaultClient }),
-        label: 'forwardTo',
         Component: function Download(props) {
           const { t } = useI18n()
           return (
             <ActionMenuItem
               onClick={props.onClick}
               left={<Icon icon={isIOSApp() ? ShareIosIcon : ReplyIcon} />}
             >
-              {t('SelectionBar.forwardTo')}
+              {t('SelectionBar.' + props.label)}
```

Comment:
We try to respect the grep test https://jamie-wong.com/2013/07/12/grep-test/

This is more easier to respect it in order to be able to remove unused translation string and so on.

Reply:
So maybe having: 
```
label: t('SelectionBar.' + props.label)
```
?

Reply:
Erf, I was looking for a line of code inside `SelectionBar`  showing that this component use the `props.label` the same way and I was forced the comply with it but I just found out that I have missread the code...

In fact it appeared that the SelectionBar "label" use `t('SelectionBar.' + actionName)`. This means that the `label` field have been never used before.

In any case I was going to say that I'm agree with you and I would like to change the way the label is generated inside the `SelectionBar` in order to make the two coherent. So let's go for you proposition and maybe in a second step we could change [this line](https://github.com/cozy/cozy-ui/blob/master/react/SelectionBar/index.jsx#L126) to use directly the `label` field. That way we have several benefices:
- We ensure that we have the same wording on both components
- We use `I18n` only once
- We make the label explicite
- We make this code grep test compliante.

Reply:
If we don't want to deduplicate the label between `Component` and `SelectionBar` we can remove the label completly.

Reply:
``` React Hook "useI18n" is called in function "move" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use"      react-hooks/rules-of-hooks```

I guess I will need to use something like:
```
label: 'SelectionBar.forwardTo', 
```
and translate inside the component